tests/lib/prepare.sh

```diff
@@ -572,22 +659,52 @@ EOF
     # - built debs
     # - golang archive files and built packages dir
     # - govendor .cache directory and the binary,
-    rsync -a -C \
+    if is_core16_system || is_core18_system; then
+        rsync -a -C \
           --exclude '*.a' \
           --exclude '*.deb' \
           --exclude /gopath/.cache/ \
           --exclude /gopath/bin/govendor \
           --exclude /gopath/pkg/ \
           /home/gopath /mnt/user-data/
+    elif is_core20_system; then
+        # prepare passwd for wormhole
+        mkdir -p /root/test-etc
+        mkdir -p /var/lib/extrausers
+        touch /var/lib/extrausers/sub{uid,gid}
+        for f in group gshadow passwd shadow; do
+            grep -v "^root:" "/etc/$f" > /root/test-etc/"$f"
+            grep "^root:" /etc/"$f" >> /root/test-etc/"$f"
```

Comment:
minor nitpick^100 the quotes are used inconsistently in this whole for loop, it would be nice to be consistent I think

Reply:
Nice catch! Fixing.